kubric/renderer/blender.py

```diff
@@ -36,6 +36,17 @@
 logger = logging.getLogger(__name__)
 
 
+def add_top_level_empty_parent() -> Any:
+  """Adds an empty parent to scene and makes it the parent of all objects."""
+  parent_obj = bpy.data.objects.new("Empty", None)
```

Comment:
Could you add the asset id to the name of the empty object here? I feel that would make the resulting Blender scene easier to work with.

Reply:
Updated.